### PR TITLE
Avoid creation of deprecated LFS_CONTENT_PATH setting

### PR DESCRIPTION
**Pull request by earl-warren**
Fixes: https://codeberg.org/forgejo/forgejo/issues/818

### CODE REVIEW

File: modules/setting/lfs.go [hunk 1]
```diff
@@ -36,7 +36,9 @@ func loadLFSFrom(rootCfg ConfigProvider) {
 	// DEPRECATED should not be removed because users maybe upgrade from lower version to the latest version
 	// if these are removed, the warning will not be shown
 	deprecatedSetting(rootCfg, "server", "LFS_CONTENT_PATH", "lfs", "PATH", "v1.19.0")
-	lfsSec.Key("PATH").MustString(sec.Key("LFS_CONTENT_PATH").String())
+	if sec.HasKey("LFS_CONTENT_PATH") {
```
Comment (lunny) [inline modules/setting/lfs.go:39]: This will not create `LFS_CONTENT_PATH`.

Comment (wxiaoguang) [reply modules/setting/lfs.go:39]: Well, it creates .... that's the ini package's quirk ....

image

Comment (lunny) [reply modules/setting/lfs.go:39]: > Well, it creates .... that's the ini package's quirk ....
> 
> image

Oh, God! I previously think only `MustXXX` will create something.

**Review by lunny — APPROVED** [review]

**Comment by wxiaoguang** [comment]
But the question is: why the deprecated option should be kept? It should have been removed in 1.19.

**Review by silverwind — APPROVED** [review]

**Comment by wxiaoguang** [comment]
> But the question is: why the deprecated option should be kept? It should have been removed in 1.19.

To answer this question: it's done in

* #23911

Use a fatal error to remind users to fix their config.

**Comment by lunny** [comment]
Replaced by #23911

**Pull request closed without merging**